packages/terra-clinical-result/src/common/observation/_Observation.jsx

```diff
@@ -48,17 +43,24 @@ const defaultProps = {
   hideUnit: false,
 };
 
-const interpretationIndicators = {
-  CRITICAL: <IconCritical className={cx('icon-interpretation')} />,
-  EXTREMEHIGH: <IconCritical className={cx('icon-interpretation')} />,
-  EXTREMELOW: <IconCritical className={cx('icon-interpretation')} />,
-  PANICHIGH: <IconCritical className={cx('icon-interpretation')} />,
-  PANICLOW: <IconCritical className={cx('icon-interpretation')} />,
-  VABNORMAL: <IconCritical className={cx('icon-interpretation')} />,
-  POSITIVE: <IconCritical className={cx('icon-interpretation')} />,
-  ABNORMAL: <IconAbnormal className={cx('icon-interpretation')} />,
-  HIGH: <IconHigh className={cx('icon-interpretation')} />,
-  LOW: <IconLow className={cx('icon-interpretation')} />,
+const interpretationIndicatorMap = {
+  critical: <IconCritical className={cx('icon-interpretation')} />,
+  'critical-high': <IconCritical className={cx('icon-interpretation')} />,
+  'critical-low': <IconCritical className={cx('icon-interpretation')} />,
+  positive: <IconCritical className={cx('icon-interpretation')} />,
+  abnormal: <IconAbnormal className={cx('icon-interpretation')} />,
+  high: <IconHigh className={cx('icon-interpretation')} />,
+  low: <IconLow className={cx('icon-interpretation')} />,
+};
+
+const interpretationTextClassMap = {
+  critical: 'critical',
+  'critical-high': 'critical',
+  'critical-low': 'critical',
+  positive: 'positive',
```

Comment:
How come positive sin't using the critical class map if it uses the same icon?

Reply:
"Positive" is unique -- for the moment it shares the same visual style as Critical (same color and icon), but it is it's own standalone clinical concept and needs to be separated because it has the option to be handled differently and in some cases is a lower "priority" of a status.